splink/comparison_level_library.py

```diff
@@ -22,8 +22,12 @@ def __init__(self, col_name):
             "sql_condition": f"{col.name_l()} IS NULL OR {col.name_r()} IS NULL",
             "label_for_charts": "Null",
```

Comment:
See my comment above about removing the "column_name" from the dict. Then we adjust this to `"label_for_charts" : f"{col_name} is Null"`. And similar for `ExactMatchLevelBase`, etc.

Reply:
I've already wanted this in other contexts, so this is an improvement outside of the context of this PR